doc/interface/unit_operations/multi_channel_transport_model.rst

```diff
@@ -1,7 +1,12 @@
 .. _multi_channel_transport_model_config:
 
 Multichannel Transport model (MCT model) 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+========================================
+
+Group /input/model/unit_XXX - UNIT_TYPE = MULTI_CHANNEL_TRANSPORT
+-----------------------------------------------------------------
+
+For information on model equations, refer to :ref:`multi_channel_transport_model`.
```

Comment:
the convention (so far) is to have the suffix `_model` for the model documentation and the `_config` suffix for the interface specification. So it's not super important, and also not very pretty, but for consistency, consider renaming to `multi_channel_transport_model_model` (check also for other occurrences!)

Reply:
Good catch, I will fix that for consistency